lib/util.mjs

```diff
@@ -56,3 +57,25 @@ export function replace(values, valueProp, replacerMap) {
     }
   });
 }
+
+/**
+ * Executes command in a subprocess.
+ *
+ * @param {string} cmd The command to run in the subprocess.
+ * @param {Array<string>} args Additional args to include when running the command.
+ * @param {string} cwd The working directory in which to run the command.
+ * @param {boolean=} isNodeCmd If set, command will be run in Node.
+ * @returns A Promise containing the result of the command.
+ */
+export async function execaAndLog(cmd, args, cwd, isNodeCmd = false) {
```

Comment:
Sorry, I think I actually preferred it before. It's more verbose, but I find the `isNodeCmd` parameter confusing. You have to do things like `/*isNodeCmd=*/true`.

Reply:
Understood! I think we could avoid the inline comment by accepting a destructured object as a parameter:
```export async function execaAndLog({cmd, args, cwd, isNodeCmd = false}) { ... }```
and then call the function like:
```execaAndLog({cmd=cmd, args=args, cwd=cwd,, isNodeCmd = false}) .```

However, that's not really a pattern we use in this codebase. Instead, I wrote a higher order function to call `execa` commands with logs.